### ISSUE TITLE
Mark all non-async functions as obsolete

### ISSUE THREAD

**Issue by vanillajonathan** [FortnoxAB/csharp-api-sdk#180]
The library is basically a wrapper over `HttpClient` which itself does not support non-async functions.
Using async functions in a synchronous way is not recommended and can lead to problems.

Decorator all the non-async methods with the `Obsolete` attribute, or even better, just remove them from the code.

**Comment by richardrandak** [comment]
Agreed. The users can handle the async to sync transition on their own if they choose to. I would like to remove all the non async methods which are just duplicates.

However, it can be a big task for some users to adapt, so I think marking them obsolete first is better approach.

**Comment by vanillajonathan** [comment]
I can see it becoming a task to adapt and that developers might need some time to adapt, so marking them as obsolete would be a good first step.

I hope to see them marked as obsolete.

**Comment by vanillajonathan** [comment]
@richardrandak Consider complete removal of all non-async methods.
